pkg/eventlog/mongo/mongo.go

```diff
@@ -0,0 +1,84 @@
+package mongo
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+	"github.com/gomods/athens/pkg/eventlog"
+)
+
+// Log is event log fetched from olympus server
+type Log struct {
+	s   *mgo.Session
+	d   string // database
+	c   string // collection
+	url string
+}
+
+// NewLog creates log reader from remote log
+func NewLog(url string) (*Log, error) {
+	return NewLogWithCollection(url, "eventlog")
+}
+
+// NewLogWithCollection creates log reader from remote log
+func NewLogWithCollection(url, collection string) (*Log, error) {
+	m := &Log{
+		url: url,
+		c:   collection,
+		d:   "athens",
+	}
+	return m, m.Connect()
+}
+
+// Connect establishes a session to the mongo cluster.
+func (m *Log) Connect() error {
+	s, err := mgo.Dial(m.url)
+	if err != nil {
+		panic(err)
```

Comment:
can you return the error here? would rather do that than panic in a non-root stack frame